Associative-Containers/exercise2.cpp

```diff
@@ -1,85 +1,141 @@
-#include<map>
-#include<iostream>
-#include<numeric>
-#include<cmath>
-#include<functional>
-#include<cassert>
+#include <map>
+#include <iostream>
+#include <numeric>
+#include <cmath>
+#include <functional>
+#include <cassert>
+#include <iomanip>
+enum class ErrorCode
+{
 
-enum class ErrorCode{
-    
     Ok,
     BadCharacter,
     BadFormat,
     DivideBy0,
     SqrtOfNegativeNumber,
     ModuleOfNonIntegerValue,
-    None 
-    };
+    None
+};
+bool DivideBy0 (const std::string &input_)
```

Comment:
```suggestion
bool DivideByZero (const std::string &input_)
```

I think the digit '0' is too similar to letter 'O', hence I recommend to use word instead digit, so it can be more readable

Reply:
Uploaded